Wyam/config.wyam

```diff
@@ -1,16 +1,16 @@
-#n Wyam.Razor
-
-#a "..\Solutions\Endjin.AzureWeekly.Wyam\bin\Debug\netstandard2.0\Endjin.AzureWeekly.WyamModules.dll"
-
-#theme AzureWeekly
-
-// Customize your settings and add new ones here
-Settings[Keys.Host] = "AzureWeekly.info";
-
-string newscollectionFilePath = @"..\Artefacts\NewsletterCollections\168.json";
-
-Pipelines.Add(
-    "Newsletter",
-    NewsTopicModule(newscollectionFilePath),
-    Razor(),
-    WriteFiles());
\ No newline at end of file
+#n Wyam.Razor
+
+#a "..\Solutions\Endjin.AzureWeekly.Wyam\bin\Debug\netstandard2.0\Endjin.AzureWeekly.WyamModules.dll"
+
+#theme AzureWeekly
+
+// Customize your settings and add new ones here
+Settings[Keys.Host] = "AzureWeekly.info";
+
+string newscollectionFilePath = @"..\Artefacts\NewsletterCollections\168.json";
+
+Pipelines.Add(
+    "Newsletter",
+    NewsTopicModule(newscollectionFilePath),
+    Razor().WithLayout("/_Index.cshtml"),
+    WriteFiles((doc, ctx) => doc.Get<DateTime>("NewsletterFrom").ToString("yyyyMMdd") + ".html"));
```

Comment:
Since the documents being output by `NewsTopicModule` don't actually exist on disk and we didn't give them a source when we created them, we have to tell the `WriteFiles` module exactly where you write each one. You can do that by using the lambda constructor for the module which gives you total control. In this case, I'm outputting each document to a location equal to it's "NewsletterFrom" date (that looks like it's getting set as metadata in your `NewsTopicModule` module, but as far as I can tell, it's not currently present in the JSON so the value ends up being default). You can change this output path to whatever you need.